packages/gutenframe/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@automattic/gutenframe",
+	"version": "1.0.0-alpha.0",
+	"description": "Utilities for the block editor iframe connections",
+	"main": "dist/server.js",
+	"sideEffects": false,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/wp-calypso.git",
+		"directory": "apps/gutenframe"
```

Comment:
It's not actually _in_ `apps/gutenframe`, although it could be.

Is this more like `o2-block`, in that it produces a runnable bundle? Or is it more like the other packages, that provide something which can be imported?

`apps/` didn't exists until very recently, but we might consider moving this there.

Reply:
Seems like the `apps` dir if the right place for this package so I moved it there (2cdc03c).